<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseCameraWidget</class>
 <widget class="QWidget" name="ChooseCameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="camFeedRegion">
     <item>
      <widget class="QPushButton" name="leftButton">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="feedLabel">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rightButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="savePathRegion">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="saveImagesLabel">
       <property name="text">
        <string>Save images to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="savePathText">
       <property name="text">
        <string>/home/paol/wboard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="captureButton">
     <property name="text">
      <string>Start capture</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
